cli/tauri-bundler/src/bundle/settings.rs

```diff
@@ -95,38 +95,23 @@ pub struct PackageSettings {
   pub default_run: Option<String>,
 }
 
-/// The bundle settings of the BuildArtifact we're bundling.
+/// The Linux debian bundle settings.
 #[derive(Clone, Debug, Deserialize, Default)]
-pub struct BundleSettings {
-  /// the app's identifier.
-  pub identifier: Option<String>,
-  /// the app's icon list.
-  pub icon: Option<Vec<String>>,
-  /// the app's resources to bundle.
-  ///
-  /// each item can be a path to a file or a path to a folder.
-  ///
-  /// supports glob patterns.
-  pub resources: Option<Vec<String>>,
-  /// the app's copyright.
-  pub copyright: Option<String>,
-  /// the app's category.
-  pub category: Option<AppCategory>,
-  /// the app's short description.
-  pub short_description: Option<String>,
-  /// the app's long description.
-  pub long_description: Option<String>,
-  /// the app's script to run when unpackaging the bundle.
-  pub script: Option<PathBuf>,
+pub struct DebianSettings {
   // OS-specific settings:
   /// the list of debian dependencies.
-  pub deb_depends: Option<Vec<String>>,
+  pub depends: Option<Vec<String>>,
   /// whether we should use the bootstrap script on debian or not.
   ///
   /// this script goal is to allow your app to access environment variables e.g $PATH.
   ///
   /// without it, you can't run some applications installed by the user.
-  pub deb_use_bootstrapper: Option<bool>,
+  pub use_bootstrapper: Option<bool>,
+}
+
+/// The macOS OSX bundle settings.
+#[derive(Clone, Debug, Deserialize, Default)]
+pub struct OsxSettings {
```

Comment:
I searched a bit and it looks like the term osx isn't used anymore right? Normally, it's just called a bundle.
I think maybe we could just rename it `MacOSSetting`.

Reply:
yeah you're right

Reply:
done :P